scripts/ChangesSinceLastRelease.txt

```diff
@@ -871,8 +872,6 @@ Windows.Win32.Networking.WinSock.Apis.closesocket : [DllImport(WS2_32.dll,ExactS
 Windows.Win32.Networking.WinSock.Apis.connect : [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False),Documentation(https://docs.microsoft.com/windows/win32/api/winsock2/nf-winsock2-connect),SupportedOSPlatform(windows6.0.6000)] => [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False,SetLastError=True),Documentation(https://docs.microsoft.com/windows/win32/api/winsock2/nf-winsock2-connect),SupportedOSPlatform(windows6.0.6000)]
 Windows.Win32.Networking.WinSock.Apis.freeaddrinfo : [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False),Documentation(https://docs.microsoft.com/windows/win32/api/ws2tcpip/nf-ws2tcpip-freeaddrinfo),SupportedOSPlatform(windows8.1)] => [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False,SetLastError=True),Documentation(https://docs.microsoft.com/windows/win32/api/ws2tcpip/nf-ws2tcpip-freeaddrinfo),SupportedOSPlatform(windows8.1)]
 Windows.Win32.Networking.WinSock.Apis.getaddrinfo : [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False),Documentation(https://docs.microsoft.com/windows/win32/api/ws2tcpip/nf-ws2tcpip-getaddrinfo),SupportedOSPlatform(windows8.1)] => [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False,SetLastError=True),Documentation(https://docs.microsoft.com/windows/win32/api/ws2tcpip/nf-ws2tcpip-getaddrinfo),SupportedOSPlatform(windows8.1)]
-Windows.Win32.Networking.WinSock.Apis.gethostbyaddr : [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False),Documentation(https://docs.microsoft.com/windows/win32/api/winsock2/nf-winsock2-gethostbyaddr),Obsolete(getnameinfo() or GetNameInfoW(),SupportedOSPlatform(windows8.1)] => [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False,SetLastError=True),Documentation(https://docs.microsoft.com/windows/win32/api/winsock2/nf-winsock2-gethostbyaddr),Obsolete(getnameinfo() or GetNameInfoW(),SupportedOSPlatform(windows8.1)]
-Windows.Win32.Networking.WinSock.Apis.gethostbyname : [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False),Documentation(https://docs.microsoft.com/windows/win32/api/winsock2/nf-winsock2-gethostbyname),Obsolete(getaddrinfo() or GetAddrInfoW(),SupportedOSPlatform(windows8.1)] => [DllImport(WS2_32.dll,ExactSpelling=True,PreserveSig=False,SetLastError=True),Documentation(https://docs.microsoft.com/windows/win32/api/winsock2/nf-winsock2-gethostbyname),Obsolete(getaddrinfo() or GetAddrInfoW(),SupportedOSPlatform(windows8.1)]
```

Comment:
What are these removals? Will block just to make sure we cover this as it could impact projections negatively.

Reply:
They were superseded by other deltas. The SetLastError changes are still there.

Reply:
I'm still a bit confused, a PR should not have any removals to historical changes right? Was this a merge mishap?